yais.sh

```diff
@@ -457,9 +402,48 @@ runChartmanOperatorCommand () {
     ARGS="-d --restart unless-stopped --name $ARG_CHARTMAN_UI_CONTAINER $PORT_MAPPING -e PUBLIC_PORTS_ASSIGNMENT=$PUBLIC_PORTS_ASSIGNMENT -v $ARG_NPMRC_FILE:/root/.npmrc $COMMON_ARGS server"
   fi
 
-  docker run $ARGS
+  response=`docker run $ARGS 2>&1`
```

Comment:
I would avoid `` as much as possible))
```suggestion
  response=$(docker run $ARGS 2>&1)
```

Reply:
why do you need 2>&1 ?